lib/assessment.md

```diff
@@ -138,7 +147,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+div will be at the top left, divOne at the right and divFour will be under div on the left.
```

Comment:
All of the CSS is visible to all of the `div`s. Where will each div be positioned?